components/gripper.py

```diff
@@ -0,0 +1,33 @@
+from wpilib import DoubleSolenoid, DigitalInput, PneumaticsModuleType
+
+import ids
+
+
+class Gripper:
+    def __init__(self):
```

Comment:
Ensure all the methods are covered by the type checker.

```suggestion
    def __init__(self) -> None:
```